client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-build/windows-frameworks/windows-frameworks.component.ts

```diff
@@ -93,32 +93,35 @@ export class WindowsFramworksComponent implements OnInit, OnDestroy {
     private _cacheService: CacheService,
     private _translateService: TranslateService
   ) {
-    this.wizard.buildSettings
-      .get('pythonSettings')
-      .get('framework')
-      .valueChanges.takeUntil(this._ngUnsubscribe$)
-      .subscribe(val => {
-        this.setupValidators(val);
-      });
+    this.setupValidators();
   }
 
   get getFramework() {
     return this.selectedPythonFramework;
   }
-  private setupValidators(val) {
+  private setupValidators() {
     this.requiredValidator = new RequiredValidator(this._translateService, false);
-    if (this.wizard.wizardValues.buildSettings.applicationFramework === WebAppFramework.Python && val === PythonFrameworkType.Django) {
-      this.wizard.buildSettings
-        .get('pythonSettings')
-        .get('djangoSettingsModule')
-        .setValidators([this.requiredValidator.validate.bind(this.requiredValidator)]);
-      this.wizard.buildSettings
-        .get('pythonSettings')
-        .get('djangoSettingsModule')
-        .updateValueAndValidity();
-    } else {
-      this.removeValidators();
-    }
+    this.wizard.buildSettings.get('applicationFramework').setValidators([this.requiredValidator.validate.bind(this.requiredValidator)]);
+    this.wizard.buildSettings.get('applicationFramework').updateValueAndValidity();
+
+    this.wizard.buildSettings
+      .get('pythonSettings')
+      .get('framework')
```

Comment:
do we have a required validator for python framework. The default value seems to be empty.

Reply:
bottle is the default framework for python

Reply:
In Ux it shown empty. can you please change it to Bottle?

Reply:
Fixed it, the code was trying to put display value as the value and hence it must be coming empty.